### ISSUE TITLE
ComfyUI workflow examples

### ISSUE THREAD

**Issue by piovis2023** [aihacker111/Efficient-Live-Portrait#10]
Do you have any comfyui workflow examples?

There are quite a few variations of Live Portrait for ComfyUI.

This version seem the most promising. I'd like to see an example so I can learn it.

Thank you

**Comment by aihacker111** [comment]
@piovis2023 Please waiting, in many near day , I have problem with my new research paper , so stay tuned , I'll update big features in the next few day . Thank you for enjoying it

**Comment by piovis2023** [comment]
You EXCELLENT person.

OK looking forward to it. Good luck with your research paper. Let me know if I can help you with it at all!

**Comment by aihacker111** [comment]
@piovis2023 I'm just see ComfyUI workflow , so  I think I'll build this project as a package python and you can use it to paste into ComfyUI to run. Will have details later

**Comment by piovis2023** [comment]
Sounds great. Im excited to test and provide quick feedback and suggestions to help you perfect it!

**Comment by aihacker111** [comment]
@piovis2023 in this tonight , I'll update tensorrt on it , and tomorrow will update new feature that integrate with SadTalker and Anything-anypose model, maybe next week will integrate with controlnet-open-pose and animate-diff-motion from ByteDance

**Comment by piovis2023** [comment]
Sounds good. I'm playing around with incorporating live portrait with mimic motion at the moment.

I wonder if MusePose would be better. MusePose is just a BIG headache to install on a Windows OS due to its sensitive dependencies.

I heard TensorRT has lots of complications with diffusion models so I uninstalled it. 

I encourage you to release a workflow version without TensorRT first. I can experiment and test it for you while you try to add TensorRT if you like.

If you are successful at this, I'll give you some really good ideas on how to make your workflow game-changing!

**Comment by aihacker111** [comment]
@piovis2023 TensorRT is very Faster, so why for update latest, I'm also update for onnxruntime and tensorrt use can use both of it

**Comment by aihacker111** [comment]
@piovis2023 Also help me to clean or refactor some case in code, I'm tired for coding everyday for many project

**Comment by piovis2023** [comment]
Ah - I'm not a coder BUT I'm happy to help you find some people that might be able to do this. 

Do you want me to task on forums for you?

**Comment by aihacker111** [comment]
@piovis2023 TensorRT is worked, I'm testing on colab , so I will push it at 9:00 , you can use my colab to test it, very faster

**Comment by piovis2023** [comment]
Oh! Well done!

OK I can try Collab for you. I really need comfyui though.

**Comment by aihacker111** [comment]
@piovis2023 New version is availabled  , support for lastest cuda and tensorrt in colab , Please try it out in the colab folder notebook

**Comment by PiovisTeam** [comment]
This is great, I've just run colab. Where would I find the output please?

**Comment by aihacker111** [comment]
which command line you use to run show me it

**Comment by aihacker111** [comment]
@piovis2023 that's mean your input video can't open 
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x6670580] moov atom not found
/content/Efficient-Live-Portrait/experiment_examples/examples/driving/Lipbite_V2.mp4: Invalid data found when processing input

**Comment by aihacker111** [comment]
@piovis2023 please make sure your video has end file .mp4 and can't open in your computer and then upload on drive and change the path

**Comment by piovis2023** [comment]
Cam on @aihacker111 (just realised you are from Vietnam - one of my favourite countries in the world!!!)

Good news = The colab script ran

Bad news = Results were scary!

<img width="1280" alt="image" src="https://github.com/user-attachments/assets/4728de1b-4502-430e-9779-9b481dc801d0">



Thanks.

**Comment by aihacker111** [comment]
@piovis2023 You can try the better result by turn off fp16 using orignal fp32 in the command line remove -fp16

**Comment by piovis2023** [comment]
@aihacker111 YES!!!

Much much better.

Only a little bit of warping.

Areas of attention (bonus):
- putting tongue out

We are going to try another test video with the following
- putting head completely down and completely up
- turning 360. 

I will be interesting to explore the real limitations of your good work!

**Comment by aihacker111** [comment]
@piovis2023 Thank you, my friend, I'm take time to do new paper with latest update in tonight, stay tuned

**Comment by PiovisTeam** [comment]
@aihacker111 the 'animations' folder does not appear with the new animations anymore when the script is run. Has anything been updated/changed? It was working before.

**Comment by aihacker111** [comment]
@aihacker111 if you run on colab , refresh button in the left will be show it

**Comment by piovis2023** [comment]
Yeah @aihacker111 I'm having the same issue. Animations folder stopped appearing. I closed my computer, reloaded the same file and the animation folder didn't come up this time?

Did something change my friend?

**Comment by aihacker111** [comment]
Do you run on colab or computer

**Comment by PiovisTeam** [comment]
On colab

**Comment by aihacker111** [comment]
Maybe it is the issues with colab, im just test again , everything ok

**Comment by piovis2023** [comment]
Thanks. Will check again.

**Comment by PiovisTeam** [comment]
@aihacker111 had the same issue. It didn't produce the 'animations' folder.

**Comment by PiovisTeam** [comment]
@aihacker111 here is what's at the bottom once the driver and source image are run:

 File "/content/Efficient-Live-Portrait/run_live_portrait.py", line 27, in <module>
    main(args.video, args.image, args.run_time, args.real_time, args.half_precision)
  File "/content/Efficient-Live-Portrait/run_live_portrait.py", line 13, in main
    live_portrait = EfficientLivePortrait(use_tensorrt, half_precision, **kwargs)
  File "/content/Efficient-Live-Portrait/LivePortrait/fast_live_portrait_pipeline.py", line 12, in __init__
    super().__init__(use_tensorrt, half,  **kwargs)
  File "/content/Efficient-Live-Portrait/LivePortrait/live_portrait/portrait.py", line 12, in __init__
    self.predictor = EfficientLivePortraitPredictor(use_tensorrt, half, **kwargs)
  File "/content/Efficient-Live-Portrait/LivePortrait/commons/predictor.py", line 12, in __init__
    from .utils.tensorrt_driver import TensorRTEngine
  File "/content/Efficient-Live-Portrait/LivePortrait/commons/utils/tensorrt_driver.py", line 2, in <module>
    import pycuda.driver as cuda
  File "/usr/local/lib/python3.10/dist-packages/pycuda/driver.py", line 66, in <module>
    from pycuda._driver import *  # noqa
ImportError: libcuda.so.1: cannot open shared object file: No such file or directory

**Comment by piovis2023** [comment]
@aihacker111  I hope you did well on your project paper.

Did you make any progress on this? Or comfyui? Thanks

**Comment by piovis2023** [comment]
Hi @aihacker111 I've just back and tried your updated solution. Great you got it working. The quality isn't as good as the first time that I tried it (without fp-16).

Did you change something? How can I use the older version again?

Keep up the great work!
